.github/workflows/shellcheck.yml

```diff
@@ -7,6 +7,7 @@ on:
       - "!main" # excludes main
     paths:
       - "**.sh"
+      - "sda-admin"
```

Comment:
I wonder if this is enough, shouldn't shellcheck have been triggered by this commit? I think `tools/**` or `tools/sda-admin` might be needed, but haven't checked.

Reply:
I agree with what @pontus suggested. Better to verify it.

Reply:
I also agree that shellcheck should run on this.